sfcasts/ep1/tdd.md

```diff
@@ -1,43 +1,55 @@
+Test Driven Development or TDD. TDD is basically a buzzword that describes a
 4-step process for writing your test *before* the feature.
 
-Step 1: write a test for the feature. Step 2: run your test and watch it fail...
-since you haven't created that feature yet! Step 3: Write as little code as possible
-to get your test to pass. And Step 4: now that it's passing, refactor your code
-if needed to make it more awesome.
+Step 1: Write a test for the feature. Step 2: Run your test and watch it fail...
+since we haven't created that feature yet! Step 3: Write as little code as
+possible to get our test to pass. And Step 4: Now that it's passing, refactor
+your code if needed to make it more awesome
 
-So, to get the Small, Medium or Large text, I think we should add a new
+So, to get the Small, Medium, or Large text, I think we should add a new
 `getSizeDescription()` method to our `Dinosaur` class. *But*, remember, we're
-going to do this the TDD way, where step 1 is to write a *test* for that method...
-even though it doesn't exist yet. Yes, I know that's weird... but it's kinda awesome.
+going to do this the TDD way, where Step 1 is to write a *test* for that method...
+even though it doesn't exist yet. Yes, I know that's weird... but it's kinda awesome!
+
+## Step 1: Write a test for the Feature
 
 Add `public function` and let's first test that a dinosaur that's over 10 meters
 or greater is large. Inside, say `$dino = new Dinosaur()`, give him a name,
-let's use Big Eaty again, he's a cool dude, and set the length to 10.
+let's use Big Eaty again, since he's a cool dude, and set his length to 10.
 
-Then, `assertSame()` that `Large` will be equal to `$dino->getSizeDescription()`.
-For our failure message, lets use This is supposed to be a large Dinosaur.
+Then, `assertSame()` that `Large` will be identical to `$dino->getSizeDescription()`.
+For our failure message, let's use this is supposed to be a large Dinosaur.
 Yes, we're *literally* testing a method that doesn't exist yet. That's TDD.
 
+
+## Step 2: Run the test and watch it fail
+
 Ok, step 1 of TDD is done. Step 2 is to run our test and make sure it fails.
 Open up a terminal and then run `./vendor/bin/phpunit` and also add the
 `--testdox` flag.
 
+```terminal
+./vendor/bin/phpunit --testdox
+```
+
 And... great! 2 tests, 4 assertions, and 1 error.
-Our new test failed because, of course, we called an undefined
-method! We kind of knew
-this would happen. I'll explain why in a minute, but notice that our "this is
-supposed to be at large dino" message isn't showing up here.
+Our new test failed because, of course, we called an undefined method! We kind
+of knew this would happen. I'll explain why in a minute, but notice that our
+"this is supposed to be at large dinosaur" message isn't showing up here?
+
+## Step 3: Write simple code to make it pass
 
 Time for step 3 of TDD: write simple code to make this test pass.
 This part, taken literally, can get kinda funny. Watch:
```

Comment:
minor, on the next line, `Dinosaur` should have ticks around it because we're referring to that technical word.